### PR TITLE
define include in anonymous module

### PR DESCRIPTION
**Pull request by KristofferC**

### CODE REVIEW

File: recursively_regenerate_jlls.jl [hunk 1]
```diff
@@ -38,7 +38,9 @@ function get_json_obj(dep_name::String)
     m = Module(:__anon__)
```
Comment (fredrikekre) [inline recursively_regenerate_jlls.jl:38]: Drive-by comment here, but this should probably be `Module(gensym())`?

Comment (giordano) [reply recursively_regenerate_jlls.jl:38]: Shouldn't matter too much in this simple script, but in general probably yes, so here we go: #1684
